### DISCUSSION TITLE
open5gs docker container not zmq based like instructions say

### DISCUSSION THREAD

**Discussion by joshuamoorexyz** [srsran/srsRAN_Project#207] [General Help]
I am trying the instructions here https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html# but it seems the default configuration for the container for open5gs is using gnb_rf_b200_tdd_n78_20mhz.yml which gives the error here when following the instructions on the gnb 

joshuamoorexyz@joshuamoorexyz-B550-UD-AC:~/Github/srsRAN_Project/build/apps/gnb$ sudo ./gnb -c gnb_zmq.yaml 
Lower PHY in executor blocking mode.

--== srsRAN gNB (commit 1afd7240f) ==--

Connecting to AMF on 10.53.1.2:38412
Available radio types: uhd.
Factory for radio type zmq not found. Make sure to select a valid type.
srsGNB ERROR: Unable to create radio session.
Aborted

Trying to change it in the docker-compose file to the gnb_zmq.yaml does not fix it. Any help?

**Comment by pgawlowicz** [comment]
Hi, it seems that srsgnb cannot find zmq:
```
 Factory for radio type zmq not found. Make sure to select a valid type.
```

Please install it and build the srsgnb again.
